Fix compiling with GCC 11

Upstream:
 - bug report: https://bugreports.qt.io/browse/QTBUG-89977
 - fix in Qt 6.1: 813a928c7c3cf98670b6043149880ed5c955efb9

--- old/qtbase/src/corelib/text/qbytearraymatcher.h
+++ new/qtbase/src/corelib/text/qbytearraymatcher.h
@@ -42,6 +42,8 @@
 
 #include <QtCore/qbytearray.h>
 
+#include <limits>
+
 QT_BEGIN_NAMESPACE
 
 

Upstream fix and backports:
 - Qt 6.1: 3eab20ad382569cb2c9e6ccec2322c3d08c0f716
 - Qt 6.2: 380294a5971da85010a708dc23b0edec192cbf27
 - Qt 6.3: 2b2b3155d9f6ba1e4f859741468fbc47db09292b

--- old/qtbase/src/corelib/tools/qoffsetstringarray_p.h
+++ new/qtbase/src/corelib/tools/qoffsetstringarray_p.h
@@ -55,6 +55,7 @@
 
 #include <tuple>
 #include <array>
+#include <limits>
 
 QT_BEGIN_NAMESPACE
 
